src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25558.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue25558 : _IssuesUITest
+{
+	public override string Issue => "ImageButton dosen't scale Image correctly";
+
+	public Issue25558(TestDevice device) : base(device)
+	{
+	}
+
+	[Test]
+	[Category(UITestCategories.ImageButton)]
+	public void Issue25558VerifyImageButtonAspects()
+	{
+		App.WaitForElement("imageButton");
+		VerifyScreenshot();
```

Comment:
Pending snapshots on Mac and Windows. Already available in the latest build.

Example, Mac:
![image](https://github.com/user-attachments/assets/f8872dc5-95e7-417c-a4d3-74f87163a803)

Could you commit the images?